stories/accordion.stories.js

```diff
@@ -39,40 +39,82 @@ AccordionContainer.defaultProps = {
   alignItems: 'center'
 };
 
+const doAlert = msg => window.alert(msg); // eslint-disable-line no-alert, no-undef
```

Comment:
added so that the functionality can be tested in storybook, to differentiate button handling and accordion toggling.